Apps/Admin/Client/Pages/PatientDetailsPage.razor.cs

```diff
@@ -113,6 +117,18 @@ private void RetrievePatientData()
             {
                 this.RetrieveMessagingVerifications();
             }
+
+            DateTime? patientProfileCreatedDateTime = this.Patient?.ProfileCreatedDateTime;
+            if (patientProfileCreatedDateTime != null)
+            {
+                this.ProfileCreatedDateTime = patientProfileCreatedDateTime.Value.ToLocalTime();
```

Comment:
Lines 124 and 130.  Why are we using ToLocalTIme() to convert to PST.  Are we not supposed to use the Timezone?  

<img width="1575" alt="Screenshot 2023-05-09 at 4 26 20 PM" src="https://github.com/bcgov/healthgateway/assets/58790456/3c215b96-c7df-427b-800a-bd1590ec1743">

Reply:
Fixed